EnglishTranslation-master/ITEM.tsv

```diff
@@ -15985,7 +15985,7 @@ ITEM_20171128_015984	Primus Migatis Rod	프리머스 미간티스 로드
 ITEM_20171128_015985	Primus Pevordimas Rod	프리머스 페버디마스 로드				
 ITEM_20171128_015986	Primus Raffye Rod	프리머스 라파이 로드				
 ITEM_20171128_015987	Masinios Rod	마시니오스 로드				
-ITEM_20171128_015988	 Increases attack of the Consciousness property by 10%. {img green_up_arrow 16 16}5 {nl}Stacks up to 10 times.	 - 스킬 공격시 10% 확률로 정신 {img green_up_arrow 16 16}5 {nl}최대 10중첩				
+ITEM_20171128_015988	 - 10% chance on skill use to gain Spirit {img green_up_arrow 16 16}5{nl}Stacks up to 10 times	 - 스킬 공격시 10% 확률로 정신 {img green_up_arrow 16 16}5 {nl}최대 10중첩				
```

Comment:
I personally like SPR much better than Spirit. The term is shorter and more used in the game.